gridt/src/app/login/login.page.spec.ts

```diff
@@ -34,4 +35,10 @@ describe('LoginPage', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it("should unsubscribe all subscriptions when leaving the page", () => {
+    component['loginSubscription'] = of(true).subscribe();
+    component.ngOnDestroy();
```

Comment:
test if subscriptions are added to subscriptions[]
test if subscriptions are unsubbed on destroy
spy on extendable class